.swiftlint.yml

```diff
@@ -0,0 +1,10 @@
+disabled_rules:
+  - trailing_whitespace
```

Comment:
We don't check this in any project, do you?

In my opinion this should be disabled in every project. I don't consider it's something useful to check either. What do you think?